django/db/backends/base/schema.py

```diff
@@ -538,6 +538,14 @@ def alter_field(self, model, old_field, new_field, strict=False):
         If `strict` is True, raise errors if the old column does not match
         `old_field` precisely.
         """
+        # Don't alter if only field name has changed.
+        if (
+            old_field.name != new_field.name and
```

Comment:
Do we need to check this? We would also like to omit altering if `name` is the same.